File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Cursa.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    [Serializable]
    public class Cursa : IObject<int>
    {
        private String numeCupa;
        private int capacitateMotor;

        public override string ToString()
        {
            return base.ToString()+" "+numeCupa.ToString()+" "+capacitateMotor.ToString();
        }

        public string NumeCupa
        {
            get => numeCupa;
            set => numeCupa = value;
        }

        public int CapacitateMotor
        {
            get => capacitateMotor;
            set => capacitateMotor = value;
        }

        public Cursa(string numeCupa, int capacitateMotor)
        {
            this.numeCupa = numeCupa;
            this.capacitateMotor = capacitateMotor;
        }
        
        public Cursa()
        { 
            
        }

        public int Id { get; set; }
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/Cursa.java
package entities.domain;

public class Cursa extends Entitate<Integer> {
    String numeCupa;
    Integer capacitateMotor;

    public Cursa(String numeCupa, Integer capacitateMotor) {
        this.numeCupa = numeCupa;
        this.capacitateMotor = capacitateMotor;
    }

    public Cursa() {

    }

    @Override
    public String toString() {
        return "Curse{" +
                "numeCupa='" + numeCupa + '\'' +
                ", capacitateMotor=" + capacitateMotor +
                '}';
    }

    public String getNumeCupa() {
        return numeCupa;
    }

    public void setNumeCupa(String numeCupa) {
        this.numeCupa = numeCupa;
    }


    public Integer getCapacitateMotor() {
        return capacitateMotor;
    }

    public void setCapacitateMotor(Integer capacitateMotor) {
        this.capacitateMotor = capacitateMotor;
    }
}
